README.md

```diff
@@ -277,6 +277,11 @@ This is an Ubuntu-specific parameter. It enables the [universe](https://help.ubu
 
 A comma-separated list of the packages to be installed on the system specified via `OS`.
 
+
+##### ENABLE_UNATTENDED_INSTALLATION=false
```

Comment:
Put it between `ENABLE_NONFREE` and `ENABLE_UNIVERSE` to follow alphabetical order.